uint8 UNINIT = 0
uint8 TRANSITION_STATE_PRE_CAL = 1
uint8 IDLE = 2
uint8 POS_CONTROL = 3
uint8 VEL_CONTROL = 4
uint8 TRANSITION_STATE_SHUTTING_DOWN = 5
uint8 TRANSITION_STATE_ERR_PROCESSING = 6
uint8 FINALIZED = 7
uint8 id
string label